solutions/2.js

```diff
@@ -0,0 +1,14 @@
+const solution = (num1, num2, num3, i = 0, biggestNum) => {
```

Comment:
a better way to do this is to start `i=1`. Then base case: `if(i >= threeNums.length)`... Then compare `threeNums[i] with threeNums[i-1]`

Reply:
Okay, made the change. Thanks.